dev-docs/bidders/emtv.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: bidder
+title: EMTV
+description: Prebid EMTV Bidder Adapter
+biddercode: emtv
+usp_supported: true
+gdpr_supported: true
+coppa_supported: true
+schain_supported: true
+floors_supported: true
+media_types: banner, video, native
+multiformat_supported: will-not-bid
+pbjs: true
+pbs: true
+pbs_app_supported: true
+safeframes_ok: true
+sidebarType: 1
+---
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Description           | Example   | Type      |
+|---------------|----------|-----------------------|-----------|-----------|
+| `placementId`      | optional | Placement Id         | `'0'`    | `'string'` |
+| `endpointId`      | optional | Endpoint Id         | `'0'`    | `'string'` |
```

Comment:
As per https://github.com/prebid/prebid-server/pull/2683, either `placementId` or `endpointId` is bidder required and not both are optional. Should update description to mention this.